frontend/src/components/program/DialogActivityEdit.vue

```diff
@@ -37,7 +42,13 @@ export default {
   components: { DialogForm, DialogActivityForm },
   extends: DialogBase,
   props: {
-    scheduleEntry: { type: Object, required: true },
+    scheduleEntry: { type: Object, required: false, default: null },
```

Comment:
shouldn't that be the responsibility of the user of the component to provide an activity?
now it is possible to neither provide an activity nor a scheduleEntry. 

Now that we see that it makes more sense that the DialogActivityEdit is given an activity

Or the simpler option: in components/activity/ScheduleEntry.vue you already have a scheduleEntry. why not use this one?

Reply:
@BacLuc fixed it